<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceAssignmentWidget</class>
 <widget class="QWidget" name="DeviceAssignmentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Device Assignment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Device Assignment</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="deviceComboBox"/>
   </item>
   <item>
    <widget class="QTableWidget" name="assignmentTable"/>
   </item>
   <item>
    <widget class="QPushButton" name="assignButton">
     <property name="text">
      <string>Assign Device</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
